ablySpec/RealtimeClient.swift

```diff
@@ -127,12 +121,12 @@ class RealtimeClient: QuickSpec {
                     
                     let oldRestHost = options.restHost
                     let oldRealtimeHost = options.realtimeHost
-                    
+
                     // Change REST and realtime endpoint hosts
-                    options.environment = "test"
+                    options.environment = "eu-central-1-a-sandbox"
```

Comment:
I don't think we should do this because we don't want a hard-coded dependency on a specific datacenter. Assuming your test does not require the hosts to exist, I think `test` was better.